config/routes/api_routes.rb

```diff
@@ -6,7 +6,7 @@ def self.extended(router)
       namespace :api, path: "api/:api_version", api_version: /v[.0-9]+(-pre)?/, defaults: { format: :json } do
         resources :trainees, param: :slug, only: %i[index show update create], constraints: RouteConstraints::RegisterApiConstraint do
           scope module: :trainees do
-            # resource :withdraw, controller: :withdraw, only: :create # TODO reopen route upon completing API updates and informing vendors
+            resource :withdraw, controller: :withdraw, only: :create
```

Comment:
A big welcome to the documentation writing club 

![image](https://github.com/user-attachments/assets/1caec30b-6da7-4745-8f20-47e592d5f443)

https://register-pr-5058.test.teacherservices.cloud/api-docs/v1.0-pre/reference#code-get-info-code

Reply:
Hmmm still missing docs 

![image](https://github.com/user-attachments/assets/ff9e2df9-8fb0-4fa6-a0a3-5426a44fed11)

https://register-pr-5058.test.teacherservices.cloud/api-docs/reference